oraclebmc/version.py

```diff
@@ -1,4 +1,4 @@
 # coding: utf-8
 # Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
 
-__version__ = "1.1.1"
+__version__ = "1.1.1post01"
```

Comment:
I don't think we're going to do a release for this before our normal release cycle, at which point this would be 1.1.2. So should we just make this 1.1.2 so we don't have to update it again before next release?

Also, can you update the changelog with the change?